### PR TITLE
INTYGFV-16082: Log errors

### PR DESCRIPTION
**Pull request by SamuelTilly**

### CODE REVIEW

**Comment by ChrilleSigerhed** [comment]
If I produce a CLIENT_ERROR, the errorId presented to the user matches with the one we send to log/error. But if I block the request/take down a service, it does not:

image

File: apps/minaintyg/src/schema/error.schema.ts [hunk 1]
```diff
@@ -2,3 +2,24 @@ import { z } from 'zod'
 
 export const ErrorType = z.enum(['unknown', 'login-failed', 'unavailable', 'logged-out', 'not-found'])
 export type ErrorTypeEnum = z.infer<typeof ErrorType>
+
+export const ErrorCode = z.enum([
+  'CLIENT_ERROR',
+  'LOGIN_FAILED',
+  'UNKNOWN_INTERNAL_ERROR',
+  'FETCH_ERROR',
```
Comment (ChrilleSigerhed) [inline apps/minaintyg/src/schema/error.schema.ts:10]: Could ```FETCH_ERROR``` instead be called ```SERVICE_UNAVAILABLE_ERROR```?

**Review by ChrilleSigerhed — APPROVED** [review]
Awsome, really well done! Really good tests aswell!

File: apps/minaintyg/src/schema/error.schema.ts (hunk 1 above)
Comment (SamuelTilly) [reply apps/minaintyg/src/schema/error.schema.ts:10]: FETCH_ERROR is from redux, could remap them but would rather not if not necessary 
image

**Pull request merged**